SPECS/cloud-init/cloud-init.spec

```diff
@@ -110,13 +112,13 @@ pip3 install --upgrade %{test_pkgs}
 make check %{?_smp_mflags}
 
 %post
-%systemd_post %{cl_services}
+%{systemd_post} %{cl_services}
 
 %preun
 %systemd_preun %{cl_services}
 
 %postun
-%systemd_postun %{cl_services}
+%{systemd_postun} %{cl_services}
```

Comment:
Did you apply this change because of a linting error? Based on my experience with another PR I think this is spurious and will actually break the RPM: see https://github.com/microsoft/CBL-Mariner/pull/2442/commits/ba6dc657517bcc8f688c8ea1bd2ea2ca5a32046a

Still, thank you very much for submitting these fixes!

Reply:
The linting change here is wrong from a style PoV because system macros should not be bracketed, so I've reverted.

 I don't think it's changed anything syntactically, unlike in your case though. (I expect the excludes-bracketing in spec-cleaner needs updating to include more of the system macros.)